components/editor/property-editor.tsx

```
@@ -7,7 +7,7 @@ import React, {
     useMemo,
     useState
 } from "react"
-import { CrateVerifyContext } from "@/components/providers/crate-verify-provider"
+import { SchemaWorker } from "@/components/providers/schema-worker-provider"
 import { Skeleton } from "@/components/ui/skeleton"
 import { Error } from "@/components/error"
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)